### PR TITLE
[ONL-7403] Migrate: ec-main-container to TS

### PR DESCRIPTION
**Pull request by raulwwq0**
**Jira Ticket:** [ONL-7403](https://fxsolutions.atlassian.net/browse/ONL-7403)

Migrate ec-main-container to TS

[ONL-7403]: https://fxsolutions.atlassian.net/browse/ONL-7403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/ec-main-container/ec-main-container.vue [hunk 1]
```diff
@@ -43,23 +43,22 @@
...
-  title: {
-    type: String,
-    default: null,
-  },
-  titleIntro: {
-    type: String,
-    default: null,
-  },
+interface MainContainerProps {
+    title?: string | null,
```
Comment (wolandec) [inline src/components/ec-main-container/ec-main-container.vue:50]: Do we need here null as title is already optional? The same for titleIntro

Comment (raulwwq0) [reply src/components/ec-main-container/ec-main-container.vue:50]: yes if we want to keep the default value as `null`, otherwise is not necessary

Comment (wolandec) [reply src/components/ec-main-container/ec-main-container.vue:50]: This is the question: do we need null here, what is the difference between null and undefined for those properties?

Comment (wolandec) [reply src/components/ec-main-container/ec-main-container.vue:50]: Let's remove null here.

Comment (raulwwq0) [reply src/components/ec-main-container/ec-main-container.vue:50]: In this case it seems there are no differences, so yep we can remove null default values for those properties

**Review by wolandec — APPROVED** [review]

**Pull request merged**